src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -24,6 +30,9 @@
  */
 public class NormalizeNamesFormatter implements Formatter {
 
+    // Avoid partition where these values are contained
+    private final Collection<String> avoidTermsInLowerCase = Arrays.asList("jr", "sr", "jnr", "snr", "von", "zu", "van", "der");
```

Comment:
Why isn't this called "affixesInLowerCase"? - Why is "von", "zu", "van" and "der" an affix? I think, only jr, sr, jnr, and snr are affixes. - As far as I can guess, "van" are not affixes

When reading https://en.wikipedia.org/wiki/Suffix_(name), it seems that jr is a name suffix, isn't it?